### DISCUSSION TITLE
Mods For Satisfactory

### DISCUSSION THREAD

**Discussion by xTz11** [GameServerManagers/LinuxGSM#3664] [Q&A] [answered]
Any way to have mods on server? I have copied mods from my game mod directory to the server but it won't start, is there something else to do?  i m on debian 10. thx a lot

**Comment by johnoclockdk** [comment, marked as answer]
not at moment

**Reply by TheWebGamer** [reply]
Is there still no way to run a modded Satisfactory server?

**Reply by xTz11** [reply]
no atm , i wait it too :D

**Comment by xTz11** [comment]
ok thx

**Reply by TheWebGamer** [reply]
Still not mods we can run in LinuxGSM for Satisfactiry?

**Reply by gOOvER** [reply]
> Still not mods we can run in LinuxGSM for Satisfactiry?

no, still no mod support for servers

**Comment by xTz11** [comment]
i dunno but i think not..

**Reply by TheWebGamer** [reply]
Nope...  Only mods are run on Windows Servers only.
